docs/content/reference/cli/glooctl_debug.md

```diff
@@ -14,7 +18,9 @@ glooctl debug [flags]
 ### Options
 
 ```
-  -h, --help   help for debug
+  -d, --directory string         directory to write debug info to (default "debug")
```

Comment:
We mentioned that stdout was also an option...if they wanted stdout for the subcommands that support it, would they have to put that in this flag? or does it just only do stdout instead of writing to a directory for the ones that do stdout?

Reply:
Yeah, the ones that do stdout just do stdout and the ones that do a directory just do a directory. The logic here basically is that I'm leaving the previous commands unchanged and the new ones really have too much info to justify writing to stdout. So I was just trying to not mislead with the root cmd's description, but if we think the mention of stdout is confusing, I can take it out.

Reply:
Also, the `debug kube` command really encompasses the info in `logs` and much of the info in `yaml` but in a more readable/better labeled format (afaict `debug logs` just appends logs from several pods without any distinction), so arguably we should deprecate those commands in favor of the new ones. But I would consider that out of scope.

Reply:
Ok thanks for the explanation. I just wasn't sure if they'd have to do something extra to get stdout, but sounds like it's the default and then if they want, they can just use the --directory option, so I think it's fine.

For the `debug logs/debug yaml`, I can add it to our doc issue to check if we use those anywhere and just change them to `debug kube`.